src/main/java/ua/com/juja/microservices/keepers/slackbot/model/request/KeeperRequest.java

```diff
@@ -0,0 +1,65 @@
+package ua.com.juja.microservices.keepers.slackbot.model.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.ToString;
+
+import ua.com.juja.microservices.keepers.slackbot.exception.WrongCommandFormatException;
+import ua.com.juja.microservices.keepers.slackbot.model.SlackParsedCommand;
+import ua.com.juja.microservices.keepers.slackbot.model.dto.UserDTO;
+
+/**
+ * @author Dmitriy Lyashenko
+ */
+@Getter
+@ToString
+public class KeeperRequest {
+
+    @JsonProperty("from")
```

Comment:
Just ask you - is it necessary for this class to have `@JsonProperty` for fields?